### PR TITLE
SQL code for Sitting Job Table

### PR DESCRIPTION
**Pull request by RotendaBBD**

### CODE REVIEW

File: flyway/migrations/V5__Create_SittingJobTable.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE SittingJob (
+    PK_jobID uniqueidentifier DEFAULT NEWSEQUENTIALID() PRIMARY KEY,
+    FK_sitterID uniqueidentifier NOT NULL,
+    FK_clientID uniqueidentifier NOT NULL,
+    startDate DATE,
+    startTime TIME,
+    endDate DATE,
+    endTime TIME,
```
Comment (jennadunfordbbd) [inline flyway/migrations/V5__Create_SittingJobTable.sql:8]: Does not match ERD.

Should be:
image
StartDateTime(DATETIME type)
EndDateTime(DATETIME type)

File: flyway/migrations/V5__Create_SittingJobTable.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    jobRate INT,
```
Comment (jennadunfordbbd) [inline flyway/migrations/V5__Create_SittingJobTable.sql:9]: Does not match ERD
Name of column should be TotalCost and of the type DECIMAL.
image

File: flyway/migrations/V5__Create_SittingJobTable.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+CREATE TABLE SittingJob (
```
Comment (jennadunfordbbd) [inline flyway/migrations/V5__Create_SittingJobTable.sql:1]: Should be called BOOKINGS
See table from ERD:
image

**Review by jennadunfordbbd — CHANGES_REQUESTED** [review]

File: flyway/migrations/V5__Create_SittingJobTable.sql (hunk 3 above)
Comment (RotendaBBD) [reply flyway/migrations/V5__Create_SittingJobTable.sql:1]: I think I might have looked to the wrong ERD here.
But I will try fixing it. 
And I can see that the branch has been merged?

**Pull request merged**